augur/dates/__init__.py

```diff
@@ -1,3 +1,4 @@
+import aniso8601
```

Comment:
Will need adding to conda/docker etc etc

Reply:
Inclusion in setup.py covers the `pip install` approach for Docker. For Conda, we'll add the dependency to the Bioconda recipe as [part of the standard release process](https://github.com/nextstrain/augur/blob/6e8ba3b7867a3a38caad79f11bd35b4dd4f86a81/docs/contribute/DEV_DOCS.md#L224).